core/math/math_2d.h

```diff
@@ -162,6 +162,8 @@ struct Vector2 {
 	}
 
 	Vector2 floor() const;
+	Vector2 ceil() const;
```

Comment:
Those also need to be bound in `core/variant_call.cpp` (in `struct _VariantCall` and `register_variant_methods()`), otherwise they're not usable in scripting.